core/src/libraries/BalanceSheet.sol

```diff
@@ -26,34 +33,21 @@ struct Prices {
 /// @notice Provides functions for computing a `Borrower`'s health
 /// @author Aloe Labs, Inc.
 library BalanceSheet {
-    function isHealthy(
-        Prices memory prices,
-        Assets memory mem,
-        uint256 liabilities0,
-        uint256 liabilities1
-    ) internal pure returns (bool) {
-        (uint256 incentive1, ) = computeLiquidationIncentive(
-            mem.fixed0 + mem.fluid0C, // total assets0 at `prices.c` (the TWAP)
-            mem.fixed1 + mem.fluid1C, // total assets1 at `prices.c` (the TWAP)
-            liabilities0,
-            liabilities1,
-            prices.c
-        );
-        return isHealthy(prices, mem, liabilities0, liabilities1, incentive1);
-    }
+    using SoladyMath for uint256;
 
     function isHealthy(
         Prices memory prices,
         Assets memory mem,
         uint256 liabilities0,
-        uint256 liabilities1,
-        uint256 incentive1
+        uint256 liabilities1
     ) internal pure returns (bool) {
-        // The liquidation incentive is added to `liabilities1` because it's a potential liability, and we
-        // don't want to re-evaluate it at the probe prices (as would happen if we added it to `liabilities0`).
         unchecked {
-            liabilities0 += liabilities0 / MAX_LEVERAGE;
-            liabilities1 += liabilities1 / MAX_LEVERAGE + incentive1;
+            liabilities0 +=
+                (liabilities0 / MAX_LEVERAGE) +
+                (liabilities0.zeroFloorSub(mem.fixed0 + mem.fluid0C) / LIQUIDATION_INCENTIVE);
+            liabilities1 +=
+                (liabilities1 / MAX_LEVERAGE) +
+                (liabilities1.zeroFloorSub(mem.fixed1 + mem.fluid1C) / LIQUIDATION_INCENTIVE);
```

Comment:
Note the liquidation incentive is still a bit of a hybrid calc, because we're measuring shortfall amounts at the TWAP:

`max(0, liabilitiesX - (mem.fixedX + mem.fluidXC))`

instead of at prices A and B using something like `fluidXA` and `fluidXB`. That's okay though, because for this fix, it doesn't matter _where the amounts come from_ so long as the price we use to combine them is uniform.